jetbrains-core/src/software/aws/toolkits/jetbrains/ui/RegionSelector.kt

```diff
@@ -31,7 +29,7 @@ class RegionSelector : ComboBox<AwsRegion>() {
     }
 
     fun setRegions(regions: List<AwsRegion>) {
-        comboBoxModel.items = regions
+        comboBoxModel.replaceAll(regions)
```

Comment:
Whats the behavior change after using replaceAll?

Reply:
behavior should be the same, but we're no longer hacking in something that the `CollectionComboBoxModel` already implements

Reply:
Richard is correct